### PR TITLE
minor gpu readme update

### PR DESCRIPTION
**Pull request by zilbermanor**

### CODE REVIEW

File: demos/gpu/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following prerequisites must be met to successfully run the Horovod demo tut
 
 The following prerequisites must be met to successfully run the RAPIDS demo tutorials (**demos/rapids**):
 
-- Your environment has one or more GPUs with the [NVIDIA Pascal](https://www.nvidia.com/en-us/geforce/products/10series/architecture/) architecture or better and [compute capability](https://developer.nvidia.com/cuda-gpus) 6.0+.
+- Your environment has one or more GPUs with the [NVIDIA Pascal](https://www.nvidia.com/en-us/geforce/products/10series/architecture/) architecture or better and [CUDA](https://developer.nvidia.com/cuda-gpus) 6.0+.
```
Comment (Sharon-iguazio) [inline demos/gpu/README.md]: The phrasing in the current doc was copied from the prerequisites on the [RAPIDS getting-started page](https://rapids.ai/start.html); see the **[Prerequisites](https://rapids.ai/start.html#prerequisites)** section; (when using the direct link, you might need to scroll up a bit to see the section title and the "GPU" line):
> **GPU:** NVIDIA Pascal™ or better with **[compute capability](https://developer.nvidia.com/cuda-gpus)** 6.0+
> ...
> **CUDA: [9.2](https://developer.nvidia.com/cuda-92-download-archive) with driver v396.37+** or **[10.0](https://developer.nvidia.com/cuda-10.0-download-archive) with driver v410.48+**

=> I prefer to stick with RAPIDS's phrasing.

File: demos/gpu/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 The GPU demos are aimed at demonstrating how to run code over graphics processing units (GPUs) in the Iguazio Data Science Platform (**the platform**).
 The **demos/gpu** directory includes the following:
 
-- A **horovod** directory with applications that use Uber's [Horovod](https://eng.uber.com/horovod/) distributed deep-learning framework, which can be used to convert a single-GPU TensorFlow, Keras, or PyTorch model-training program to a distributed program that trains the model simultaneously over multiple GPUs.
+- A **horovod** directory with applications that use Uber's [Horovod](https://eng.uber.com/horovod/) [(github)](https://github.com/horovod/horovod) distributed deep-learning framework, which can be used to convert a single-GPU TensorFlow, Keras, or PyTorch model-training program to a distributed program that trains the model simultaneously over multiple GPUs.
     The objective is to speed up your model training with minimal changes to your existing single-GPU code and without complicating the execution.
     Horovod code can also run over CPUs with only minor modifications.
```
Comment (Sharon-iguazio) [inline demos/gpu/README.md]: I recommend keeping lines 15–17 as-is and adding the following:
```
For more information and examples, see the [Horovod GitHub repository](https://github.com/horovod/horovod).
```
[HTML output — "For more information and examples, see the [Horovod GitHub repository](https://github.com/horovod/horovod)."]

Comment (zilbermanor) [reply demos/gpu/README.md]: Approved

File: demos/gpu/README.md (hunk 1 above)
Comment (zilbermanor) [reply demos/gpu/README.md]: Accepted

File: demos/gpu/README.md (hunk 2 above)
Comment (Sharon-iguazio) [reply demos/gpu/README.md]: UPDATE: I implemented the suggested changes in PR #152.

**Pull request closed without merging**